source/uwp/Renderer/lib/AdaptiveSpacingConfig.cpp

```diff
@@ -0,0 +1,91 @@
+#include "pch.h"
+#include "AdaptiveSpacingConfig.h"
+
+using namespace Microsoft::WRL;
+using namespace ABI::AdaptiveCards::XamlCardRenderer;
+
+namespace AdaptiveCards { namespace XamlCardRenderer
+{
+    HRESULT AdaptiveSpacingConfig::RuntimeClassInitialize() noexcept try
+    {
+        return S_OK;
+    } CATCH_RETURN;
+
+
+    HRESULT AdaptiveSpacingConfig::RuntimeClassInitialize(SpacingConfig spacingConfig) noexcept
+    {
+        m_sharedSpacingConfig = spacingConfig;
+        return S_OK;
+    }
+
+    _Use_decl_annotations_
+    HRESULT AdaptiveSpacingConfig::get_Small(UINT32* smallSpacing)
+    {
+        *smallSpacing = m_sharedSpacingConfig.smallSpacing;
+        return S_OK;
+    }
+
+    _Use_decl_annotations_
+    HRESULT AdaptiveSpacingConfig::put_Small(UINT32 smallSpacing)
+    {
+        m_sharedSpacingConfig.smallSpacing = smallSpacing;
+        return S_OK;
+    }
+
+    _Use_decl_annotations_
+        HRESULT AdaptiveSpacingConfig::get_Default(UINT32* smallSpacing)
+    {
+        *smallSpacing = m_sharedSpacingConfig.defaultSpacing;
+        return S_OK;
+    }
+
+    _Use_decl_annotations_
+        HRESULT AdaptiveSpacingConfig::put_Default(UINT32 smallSpacing)
+    {
+        m_sharedSpacingConfig.smallSpacing = smallSpacing;
+        return S_OK;
+    }
+
+    _Use_decl_annotations_
+    HRESULT AdaptiveSpacingConfig::get_Medium(UINT32* mediumSpacing)
+    {
+        *mediumSpacing = m_sharedSpacingConfig.mediumSpacing;
+        return S_OK;
+    }
+
+    _Use_decl_annotations_
+    HRESULT AdaptiveSpacingConfig::put_Medium(UINT32 mediumSpacing)
+    {
+        m_sharedSpacingConfig.mediumSpacing = mediumSpacing;
+        return S_OK;
+    }
+
+    _Use_decl_annotations_
+    HRESULT AdaptiveSpacingConfig::get_Large(UINT32* largeSpacing)
+    {
+        *largeSpacing = m_sharedSpacingConfig.largeSpacing;
+        return S_OK;
+    }
+
+    _Use_decl_annotations_
+    HRESULT AdaptiveSpacingConfig::put_Large(UINT32 largeSpacing)
+    {
+        m_sharedSpacingConfig.largeSpacing = largeSpacing;
+        return S_OK;
+    }
+
+    _Use_decl_annotations_
+        HRESULT AdaptiveSpacingConfig::get_ExtraLarge(UINT32* largeSpacing)
+    {
+        *largeSpacing = m_sharedSpacingConfig.largeSpacing;
```

Comment:
>largeSpacing [](start = 46, length = 12)

ExtraLarge

Reply:
Good catch

---
In reply to: [138419031](https://github.com/Microsoft/AdaptiveCards/pull/620#discussion_r138419031) [](ancestors = 138419031)